src/MaaCore/Task/Miscellaneous/StartGameTaskPlugin.cpp

```diff
@@ -9,48 +9,53 @@ bool StartGameTaskPlugin::_run()
     if (m_client_type.empty()) {
         return false;
     }
+    // check for MAC / iOS
+    if (ctrler()->get_controller_type() == ControllerType::MacPlayTools) {
+        return ctrler()->start_game(m_client_type);
+    }
+    else {
```

Comment:
The else can be omitted here, right? That's too many layers of indentation.

Reply:
Oh you're right, there's a return either way